pdsfile/cfg.py

```diff
@@ -0,0 +1,52 @@
+################################################################################
+# Store global variables that will be modified or have different values in both
+# pds3 & pds4 subclasses
+################################################################################
+
+import os
+import re
+
+PDS_HOLDINGS = 'holdings'
```

Comment:
I'm really confused about this "cfg.py" file. This is a single module, so if you override any of the variables in it, they are changed globally. Doesn't this mean that you can't have both a Pds3File and Pds4File class instantiated at the same time, because importing each will overwrite the same variables?

I think all of these variables need to be class attributes inside of PdsFile. Then you don't need the cfg.py file at all.

In fact I think we really need to get away from having any global variables at all at this point, except maybe for cache management.

Reply:
cfg.py is removed